localstack-core/localstack/services/kms/models.py

```diff
@@ -323,6 +337,15 @@ def __init__(
         self.rotation_period_in_days = 365
         self.next_rotation_date = None
 
+        if self.supports_rotation():
+            initial_rotation = KeyRotationEntry(
+                key_id=self.metadata["Arn"],
+                key_material_state="CURRENT",
```

Comment:
Instead of using hardcoded values, import the class defined in `__init__.py` file which already includes the types you have defined here. 
The file is automatically generated, you can find more details here: https://github.com/localstack/localstack/blob/main/docs/localstack-concepts/README.md